src/main.zig

```diff
@@ -2518,7 +2518,7 @@ fn buildOutputType(
             switch (c_out_mode orelse .link) {
                 .link => {
                     create_module.opts.output_mode = if (is_shared_lib) .Lib else .Exe;
-                    emit_bin = if (out_path) |p| .{ .yes = p } else EmitBin.yes_a_out;
+                    emit_bin = if (out_path) |p| .{ .yes = p } else emit_bin;
```

Comment:
`yes_a_out` looks unused after this and could be deleted.

Reply:
Ah I was on the wrong branch and was getting the old behavior.

Why was this changed?